src/psbt.cpp

```diff
@@ -267,6 +268,12 @@ void PSBTOutput::Merge(const PSBTOutput& output)
     if (m_tap_internal_key.IsNull() && !output.m_tap_internal_key.IsNull()) m_tap_internal_key = output.m_tap_internal_key;
     if (m_tap_tree.has_value() && !output.m_tap_tree.has_value()) m_tap_tree = output.m_tap_tree;
 }
+
+bool PSBTInputSigned(const PSBTInput& input, const CScript& script_pubkey)
+{
+    return VerifyScript(input.final_script_sig, script_pubkey, &input.final_script_witness, STANDARD_SCRIPT_VERIFY_FLAGS, DUMMY_SIGNATURE_CREATOR.Checker());
```

Comment:
Is having the dummy checker correct? While this will check the script, it will also pass any signature as valid.

Reply:
sounds like I need tests eh

Reply:
suggested alternative?